### ISSUE TITLE
DstScanlineSize?

### ISSUE THREAD

**Issue by Ziriax** [avaneev/avir#14]
First of all thanks for this amazing project!

I am doing my first tests with it, and I was wondering why the `resizeImage` method has a `SrcScanlineSize` arguments, but not a `DstScanlineSize`?

Best regards,
Peter

**Comment by avaneev** [comment]
It was designed to produce a continuous image, however it may take a discontinous image as an option. I do not think even this option is much needed nowadays.

**Comment by sotex** [comment]
> 
> 
> It was designed to produce a continuous image, however it may take a discontinous image as an option. I do not think even this option is much needed nowadays.

I also encountered this problem. I intended to place a thumbnail of an image in its inner area, but I couldn't do it without `DstScanlineSize`.

![图片](https://user-images.githubusercontent.com/16911453/89421643-e3ff0b00-d766-11ea-8b0c-36cee0eb25d2.png)

I hope it occupies the upper left corner, but because the `DstScanlineSize` cannot be set, it can only be like this. The target is equivalent to interlaced in two areas.
